tests/tools/oesign/test-enclave/enclave/enc.c

```diff
@@ -54,20 +68,50 @@ bool is_test_signed()
     return is_test_signed;
 }
 
+static void* _find_claim(
+    const oe_claim_t* claims,
+    size_t claims_size,
+    const char* name)
+{
+    for (size_t i = 0; i < claims_size; i++)
+    {
+        if (strcmp(claims[i].name, name) == 0)
+            return claims[i].value;
+    }
+    return NULL;
+}
+
+void formatted_string_from_uuid(
+    uint8_t* uuid_id,
+    size_t uuid_id_size,
+    char* formatted_str_id)
+{
```

Comment:
Suggest passing in size of formatted_str_id. The function is not supposed to know how big the output buffer is.